src/main/python/apache/aurora/executor/bin/thermos_executor_main.py

```diff
@@ -54,7 +54,7 @@
 
 app.configure(debug=True)
 LogOptions.set_simple(True)
-LogOptions.set_disk_log_level('DEBUG')
+LogOptions.set_disk_log_level('INFO')
```

Comment:
Would it make sense to leave the default as `DEBUG` for the sake of backwards compat, and use the new option to override it?

Reply:
+1 please

Reply:
```
from twitter.common.log.options import LogOptions
```
I'm sorry I can not use this well.
Can you give me a hint of what to do?

Reply:
I think during `initialize(options)` you can use the options.disk_log_level that you declared.

```
LogOptions.set_disk_log_level(options.disk_log_level)
```